giskard/llm/talk/config.py

```diff
@@ -0,0 +1,154 @@
+from enum import Enum
+
+from sklearn.metrics import classification_report  # Classification metrics.
+from sklearn.metrics import (  # Regression metrics.
+    explained_variance_score,
+    mean_absolute_error,
+    mean_squared_error,
+    r2_score,
+)
+
+LLM_MODEL = "gpt-4-turbo-preview"
```

Comment:
I think this is an issue to force the usage of the model, should be done in the same way as in llm / __init__.py (or client), using getenv on some env var.

Also, is it needed ? or can't the default used model be changed ?

Reply:
@kevinmessiaen @mattbit do you think, that we could use the `gpt-4-turbo-preview` as a default model? Or should I let user to create the env variable with llm version, when using the "talk" functionality?

Reply:
Done